adlfs/tests/conftest.py

```diff
@@ -59,9 +59,13 @@ def storage(host):
 
 @pytest.fixture(scope="session")
 def event_loop():
-    loop = asyncio.get_event_loop()
-    yield loop
-    loop.close()
+    policy = asyncio.get_event_loop_policy()
+    loop = policy.new_event_loop()
+    try:
+        yield loop
+    finally:
+        loop.close()
+        policy.set_event_loop(loop)
```

Comment:
with pytest-asyncio`mode=auto` we need to explicitly update the policy eventloop after closing otherwise `asyncio` may overwrite it: https://github.com/pytest-dev/pytest-asyncio/issues/257